### PR TITLE
Update TC_RR_1_1.py to match test spec changes.

### PR DESCRIPTION
**Pull request by cletnick**
The following fixes were applied:

- Step 1 was updated to remove any existing extra fabrics. (https://github.com/CHIP-Specifications/chip-test-plans/pull/2370)
- - This requires reading back the index values since they vary as fabrics are added and removed.
- Step 12 varied from the spec for the verified value.

### CODE REVIEW

**Review by cecille — APPROVED** [review]
Not sure how much it matters, but step 10 in the test plan specifies reading from ServerList instance of the descriptor cluster, so the test here doesn't match the test plan, even though it does the same thing.

**Comment by cecille** [comment]
I pulled the logs from the failing darwin test, and it seems to be a failure related to timing, not to your test. It's really hard to parse those logs to tell what failures are actuallly failures, but I THINK the errors are from

```
Test Suite 'MTRXPCListenerSampleTests' failed at 2023-01-27 02:52:47.098.
	 Executed 16 tests, with 1 failure (0 unexpected) in 20.600 (20.605) seconds
```

```
/Users/runner/work/connectedhomeip/connectedhomeip/src/darwin/Framework/CHIPTests/MTRXPCListenerSampleTests.m:763: error: -[MTRXPCListenerSampleTests test004_Subscribe] : Asynchronous wait failed: Exceeded timeout of 3 seconds, with unfulfilled expectations: "receive OnOff attribute report".
Test Case '-[MTRXPCListenerSampleTests test004_Subscribe]' failed (8.169 seconds).
```

Anyway, I just kicked it again to see if it was a flake.

**Review by woody-apple — APPROVED** [review]
